opentelemetry-sdk/src/trace/config.rs

```diff
@@ -129,9 +129,9 @@ impl Default for Config {
                     if let Some(r) = ratio {
                         Box::new(Sampler::TraceIdRatioBased(r))
                     } else {
-                        handle_error(
-                            Error::Other(String::from(
-                                "Missing or invalid OTEL_TRACES_SAMPLER_ARG value. Falling back to default: 1.0"))
+                        otel_warn!(
+                            name: "TracesSampler.TraceIdRatio.Error",
```

Comment:
the name make it sound like there is something wrong with TraceSampler, but the issue is with invalid configuration.

Reply:
done